flake.nix

```diff
@@ -0,0 +1,24 @@
+{
+  description = "rhine";
+  nixConfig.bash-prompt = "\[rhine\]$ ";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    haskell-flake-utils.url = "github:ivanovs-4/haskell-flake-utils";
+    haskell-flake-utils.inputs.nixpkgs.follows = "nixpkgs";
+    haskell-flake-utils.inputs.flake-utils.follows = "flake-utils";
+
+  };
+
+outputs = { self, nixpkgs, flake-utils, haskell-flake-utils, ... }@inputs:
+  flake-utils.lib.eachSystem [ "x86_64-linux" "aarch64-darwin" ] (system:
+    inputs.haskell-flake-utils.lib.simpleCabalProject2flake {
+      inherit self nixpkgs system;
+
+      name = "rhine";
+      packageNames = ["rhine-gloss" "rhine-examples"];
```

Comment:
Why not `"rhine"` as well? Also, could we get those names by reading the file system? Using `builtins.readDir` or similar?

Reply:
Yes good question. I thought `"rhine"` was implied since `packageNames` is optional. But I'll double check.

Reply:
Yes indeed `"rhine"` [is added to the list of packages](https://github.com/ivanovs-4/haskell-flake-utils/blob/master/simpleCabalProject2flake.nix#L75):
```
 packageNames_ = pkgs: pkgs.lib.lists.unique ([ name ] ++ packageNames);
```